[2025-03-18T15:17:56.774+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-18T15:17:56.790+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.scrap_new_data scheduled__2025-03-16T13:30:00+00:00 [queued]>
[2025-03-18T15:17:56.798+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.scrap_new_data scheduled__2025-03-16T13:30:00+00:00 [queued]>
[2025-03-18T15:17:56.798+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-03-18T15:17:56.807+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): scrap_new_data> on 2025-03-16 13:30:00+00:00
[2025-03-18T15:17:56.815+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=110) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-18T15:17:56.816+0000] {standard_task_runner.py:64} INFO - Started process 124 to run task
[2025-03-18T15:17:56.817+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'scrap_new_data', 'scheduled__2025-03-16T13:30:00+00:00', '--job-id', '1690', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmpmn2b1eqw']
[2025-03-18T15:17:56.818+0000] {standard_task_runner.py:91} INFO - Job 1690: Subtask scrap_new_data
[2025-03-18T15:17:56.856+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.scrap_new_data scheduled__2025-03-16T13:30:00+00:00 [running]> on host b7aba691e1ff
[2025-03-18T15:17:56.902+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='scrap_new_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T13:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T13:30:00+00:00'
[2025-03-18T15:17:56.904+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-18T15:18:02.144+0000] {warnings.py:112} WARNING - /opt/***/dags/Stock_pred.py:65: GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 65 of the file /opt/***/dags/Stock_pred.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.

  soup = BeautifulSoup(html)

[2025-03-18T15:18:02.184+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-18T15:18:02.184+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-18T15:18:02.187+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=scrap_new_data, run_id=scheduled__2025-03-16T13:30:00+00:00, execution_date=20250316T133000, start_date=20250318T151756, end_date=20250318T151802
[2025-03-18T15:18:02.197+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-18T15:18:02.208+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-18T15:18:02.209+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
